acceptance/bundle/templates/default-sql/script

```diff
@@ -6,3 +6,4 @@ trace $CLI bundle validate -t prod
 
 # Do not affect this repository's git behaviour #2318
 mv .gitignore out.gitignore
+mv .databricks/.gitignore .databricks/out.gitignore
```

Comment:
I don't understand the nuance -- do you want to get coverage for `.databricks` being excluded exclusively for this template?

Reply:
This records .databricks/.gitignore so we can see what is this. It's the same every time, so no need to record in all tests. default-sql is selected randomly.

Reply:
Could you add a comment highlighting this, so it is clear it doesn't need to be copy-pasted?